### ISSUE TITLE
Blind and Visually Impaired Visitors’ Experiences in Museums: Increasing Accessibility through Assistive Technologies

### ISSUE THREAD

**Issue by seita-kayukawa** [Wotipati/Blind_Accessibility_Papers#435]
### Links
- Paper: https://cgscholar.com/bookstore/works/blind-and-visually-impaired-visitors-experiences-in-museums

### Abstract
People with visual impairments generally experience many barriers when visiting museum exhibitions, given the ocular centricity of these institutions. The situation is worsened by a frequent lack of physical, intellectual and sensory access to exhibits or replicas, increased by the inaccessibility to use ICT-based local or general alternative or augmentative communication resources that can allow different interactions to sighted visitors. Few studies analyze applications of assistive technologies for multisensory exhibit design and relate them with visitors’ experiences. This article aims to contribute to the field of accessibility in museums by providing an overview of the experiences and expectations of blind and visually impaired patrons when visiting those places, based on a literature review. It also surveys assistive technologies used to enhance the experiences of visitors with vision loss while visiting museum exhibitions and spaces. From this, it is highlighted that adopting hybrid technological approaches, following universal design principles and collaborating with blind and visually impaired people, can contribute to integrate access across the continuum of visits.

<img width="801" alt="ScreenShot 2021-10-02 16 04 56" src="https://user-images.githubusercontent.com/83838079/135707120-c6b61090-dbb5-4322-9421-63d5cea22500.png">
<img width="785" alt="ScreenShot 2021-10-02 16 05 00" src="https://user-images.githubusercontent.com/83838079/135707124-a42f8d05-6982-4dc0-80e3-20c9a2076d09.png">
<img width="804" alt="ScreenShot 2021-10-02 16 05 05" src="https://user-images.githubusercontent.com/83838079/135707125-eaad11ea-9023-48aa-8cf2-4023a2c656cf.png">
<img width="791" alt="ScreenShot 2021-10-02 16 05 10" src="https://user-images.githubusercontent.com/83838079/135707127-b3691bd3-2b2d-4812-85f7-54b08ab24c46.png">
<img width="811" alt="ScreenShot 2021-10-02 16 05 20" src="https://user-images.githubusercontent.com/83838079/135707128-6a19680d-6740-44d8-9b8e-e4dab9cfc823.png">
<img width="797" alt="ScreenShot 2021-10-02 16 05 26" src="https://user-images.githubusercontent.com/83838079/135707131-7e3d142c-2a24-4ae3-877e-efa96b5afea7.png">
<img width="821" alt="ScreenShot 2021-10-02 16 05 29" src="https://user-images.githubusercontent.com/83838079/135707133-1db181a9-58fa-460b-891e-1a904d75d15a.png">

- museumを視覚障害者にとってアクセシブルにするための技術のサーベイ論文。
